airflow/www/templates/airflow/gantt.html

```diff
@@ -23,6 +23,9 @@
     href="{{url_for('appbuilder.static',filename='datepicker/bootstrap-datepicker.css')}}" >
 <link type="text/css" href="{{ url_for('static', filename='css/gantt.css') }}" rel="stylesheet" />
 <link type="text/css" href="{{ url_for('static', filename='css/tree.css') }}" rel="stylesheet" />
+  <meta name="dag-id" content="{{ dag.dag_id }}">
+  <meta name="data" content="{{ data }}">
```

Comment:
Style nit: inconsistent indenation.

Reply:
Previously we had `| safe` filter on this - do we still need it? I assume "data" is a big blob of json that includes `"` characters? Does this all come through okay?

Reply:
There is no consequence in any file. I plan to improve it in the next change. Now there is no consistency, so you can not afford it.

Everything is displayed correctly. Jinja is intelligent enough to read our intentions.

Reply:
The lines above it have an indent of 0 - the two lines have an indent of 2. (Similarly for all the other `<meta>` tags you've added.

Reply:
I updated a PR.